src/components/EventEdit.tsx

```diff
@@ -33,6 +33,8 @@ import * as EteSync from 'etesync';
 import { getCurrentTimezone } from '../helpers';
 
 import { EventType, timezoneLoadFromName } from '../pim-types';
+import RRuleEteSync, { RRuleOptions } from '../widgets/RRule';
```

Comment:
Remove the extra line you added, and probably just call it RRule, whit call it RRuleEteSync?

Reply:
its RRuleEteSync because the when we started we used the rrule.js library so it had a name conflict with RRule. but now i can change it to just RRule

Reply:
Gotcha. Yeah, please fix it.